src/i18n/locales/en.json

```diff
@@ -518,6 +518,8 @@
         "playButtonBehavior_optionAddLast": "$t(player.addLast)",
         "playButtonBehavior_optionAddNext": "$t(player.addNext)",
         "playButtonBehavior_optionPlay": "$t(player.play)",
+        "playerAlbumArtResolution": "Player Album Art Resolution",
+        "playerAlbumArtResolution_description": "The resolution for the large player's album art preview. Larger makes it look more crisp, but may slow loading down. Defaults to 0, meaning auto",
```

Comment:
These need to be in lowercase similar to the other translation strings since we're applying postprocessing on them in the components.

Reply:
Never realized that the rest of them were actually in all lowercase when I was making it. Also adjusted my old button size PR to be lowercase as well as kgarner's cache clearing strings (which I can revert if those changes are unwanted).